01-CodeReviewEx/Shop-Ex/Account.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shop_Ex
+{
+    public class Account
+    {
+        public double principal, rate; int daysActive, accountType;
```

Comment:
Deklaracja kolejnych zmiennych powinna odbywać się w kolejnych, oddzielnych liniach kodu, sugeruję rozbicie z uwzględnieniem specyfikatorów dostępu.  To samo dla fragmentu gdzie są deklarowane zmienne innego typu (int).

Reply:
Dodatkowo wydaje mi się, że zmienne powinny mieć specyfikator dostępu "private". W tym przypadku sugeruję dodanie konstruktora oraz getterów/setterów